### PR TITLE
Add order param for pagination

### PR DESCRIPTION
**Pull request by pliao28**

### CODE REVIEW

File: src/WorkOS.net/Services/_common/_interfaces/ListOptions.cs [hunk 1]
```diff
@@ -24,5 +24,11 @@ public class ListOptions : BaseOptions
         /// </summary>
         [JsonProperty("limit")]
         public long? Limit { get; set; }
+
+        /// <summary>
+        /// Order of records returned by ascending or descending order.
+        /// </summary>
+        [JsonProperty("order")]
+        public string Order { get; set; }
```
Comment (maxdeviant) [inline src/WorkOS.net/Services/_common/_interfaces/ListOptions.cs:32]: Could we use a more strongly typed representation by making this an enum?

```cs
[JsonConverter(typeof(StringEnumConverter))]
public enum OrderBy
{
    [EnumMember(Value = "asc")]
    Asc,
    [EnumMember(Value = "desc")]
    Desc
}
```

I can't remember if enums are value types, so we would either have this:

```suggestion
        public PaginationOrder? Order { get; set; }
```

Of if they are reference types then we won't be able to mark it as optional (without nullable reference types, which we don't currently have enabled):

```suggestion
        public PaginationOrder Order { get; set; }
```

Comment (pliao28) [reply src/WorkOS.net/Services/_common/_interfaces/ListOptions.cs]: Looks like they are value types:
 https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/enum
 
 Added as enum!

File: src/WorkOS.net/Services/_common/_interfaces/ListOptions.cs [hunk 2]
```diff
@@ -24,5 +24,11 @@ public class ListOptions : BaseOptions
         /// </summary>
         [JsonProperty("limit")]
         public long? Limit { get; set; }
+
+        /// <summary>
+        /// Order of records returned by ascending or descending order.
+        /// </summary>
+        [JsonProperty("order")]
+        public PaginationOrder? Order { get; set; }
```
Comment (hadihallak) [inline src/WorkOS.net/Services/_common/_interfaces/ListOptions.cs:32]: similar to typescript, the question mark here marks it as optional right?

Comment (pliao28) [reply src/WorkOS.net/Services/_common/_interfaces/ListOptions.cs:32]: Correct!

**Review by hadihallak — APPROVED** [review]

**Review by maxdeviant — APPROVED** [review]

**Pull request merged**